docs/metrics.md

```diff
@@ -93,6 +93,9 @@ Virtual Machine last transition timestamp to migrating status. Type: Counter.
 ### kubevirt_vm_non_running_status_last_transition_timestamp_seconds
 Virtual Machine last transition timestamp to paused/stopped status. Type: Counter.
 
+### kubevirt_vm_resource_requests
+Resources requested by Virtual Machine. Type: Gauge.
```

Comment:
@machadovilaca  Can you please mention these are CPU and memory requests?

Reply:
cpu will only be added in https://github.com/kubevirt/kubevirt/pull/12625